src/pcre2_intmodedep.h

```diff
@@ -729,8 +729,11 @@ typedef struct named_group {
 of translating META code to byte code. */
 
 typedef struct class_ranges {
-  struct class_ranges *next;  /* Next class ranges */
-  size_t range_list_size;     /* Size of ranges array */
+  struct class_ranges *next;       /* Next class ranges */
+  uint16_t range_list_size;        /* Size of ranges array */
+  uint16_t char_lists_types;       /* Bit set for char lists. */
```

Comment:
move the uint16_t at the end of the struct to avoid unnecessary padding

Reply:
I will, although in this particular case, I just move the padding at the end.